docs/en/setup/backend/backend-zabbix.md

```diff
@@ -0,0 +1,67 @@
+# Zabbix Receiver
+Zabbix receiver is accepting the metrics of [Zabbix Agent Active Checks protocol](https://www.zabbix.com/documentation/current/manual/appendix/items/activepassive#active_checks) fromat into the [Meter System](./../../concepts-and-designs/meter.md).
+
+## Module define
+```yaml
+receiver-zabbix:
+  selector: ${SW_RECEIVER_METER:default}
+  default:
+    # Export tcp port, Zabbix agent could connected and transport data
+    port: 13800
+    # Enable config when receive agent request
+    activeFiles: agent.yaml
```

Comment:
Please take a reference of OTEL rules, I remember it uses file name without `.yaml`.